easybuild/easyconfigs/q/QCxMS/QCxMS-5.1.3.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'QCxMS'
+version = '5.1.3'
+
+homepage = 'https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html'
+description = """QCxMS is a quantum chemical based program to calculate electron ionization (EI)
+ and collision induced dissociation (CID) mass spectra using Born-Oppenheimer Molecular Dynamics
+ (BO-MD). It is the successor of the QCEIMS program, in which the EI part is exchanged to x to
+ account for the greater general applicibility of the program.
+"""
+
+toolchain = SYSTEM
+
+# current release does not include source code
+source_urls = ['https://github.com/qcxms/QCxMS/releases/download/v.%(version)s']
+sources = ['%(name)s.v.%(version)s.tar.xz']
+checksums = ['3522922d1da1fa449af5955fba6744b3e3338b5bae26ad3e9846bf6b96136ffa']
+
+local_bins = ['pqcxms', 'qcxms', 'q-batch', 'getres']   # 5.1.3 doesn't include 'plotms'
+preinstall_cmd = "mkdir bin && mv %s bin" % " ".join(local_bins)
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_bins],
+    'dirs': [],
+}
+
```

Comment:
@SebastianAchilles though `qcxms` doesn't seem to have a `--help` or `--version`, just running it without arguments does print some output, and returns an exit code 0. So I guess we could use that as a sanity check command?